static/src/javascripts-legacy/projects/common/views/svgs.js

```diff
@@ -145,10 +143,11 @@ define([
     };
 
     return function (name, classes, title) {
-        if (config.switches.webpack) {
+        // #wp-rjs - needed by karma
+        if (svgs[name].markup) { // webpack way
```

Comment:
I think this test would make better sense inside the `svg` function, just in case someone uses `svg` directly rather than `svgs`. Also, since we're going 100% webpack, is it even still necessary? Cannot we assume the `markup` will exist all the time?